docs/bokeh/source/docs/user_guide/server/library.rst

```diff
@@ -1,7 +1,7 @@
 .. _ug_server_library:
 
-Embedding Bokeh server as a library
-===================================
+Bokeh server APIs
+=================
```

Comment:
cc @tcmetzger I actually went this this which I think is better than all the other options I had suggested before